src/cubi_tk/snappy/pull_sheets.py

```diff
@@ -137,11 +137,11 @@ def setup_argparse(parser: argparse.ArgumentParser) -> None:
     )
 
     parser.add_argument(
-        "--assay-txt",
+        "--assay-uuid",
         default=None,
         required=False,
         type=str,
-        help="Assay Name e.g. 'a_project_exome_sequencing.txt' if multiple assays are present",
+        help="Assay UUID for assay if multiple assays are present",
     )
```

Comment:
This will apply to ALL projects defined in the snappy config, however it will only make sense to one.
Maybe add a warning somewhere if multiple projects are foud while assay_uuid is defined?